From c2578aa8aeee50dc5f4dd9099b7581f5f3b85570 Mon Sep 17 00:00:00 2001
From: Alexey Minnekhanov <alexeymin@postmarketos.org>
Date: Thu, 18 Sep 2025 20:52:46 +0300
Subject: [PATCH] dt-bindings: mfd: qcom,spmi-pmic: Add pmi8998-fg to
 fuel gauge options

Add newly added PMI8998 battery fuel gauge driver to the list of allowed
subnodes of qcom SPMI PMICs. This fixes lots of dtbs_check errors.

Signed-off-by: Alexey Minnekhanov <alexeymin@postmarketos.org>

Source-Commit: https://github.com/sdm660-mainline/linux/commit/c2578aa8aeee50dc5f4dd9099b7581f5f3b85570
Scope: PMIC-common
Porting notes: Original PMIC binding extension applies unchanged to Linux 7.3-rc5.
---
diff --git a/Documentation/devicetree/bindings/mfd/qcom,spmi-pmic.yaml b/Documentation/devicetree/bindings/mfd/qcom,spmi-pmic.yaml
index 135ebcb0d00d2a..a6c8a1897cef94 100644
--- a/Documentation/devicetree/bindings/mfd/qcom,spmi-pmic.yaml
+++ b/Documentation/devicetree/bindings/mfd/qcom,spmi-pmic.yaml
@@ -152,6 +152,7 @@ patternProperties:
     type: object
     oneOf:
       - $ref: /schemas/power/supply/qcom,pm8916-bms-vm.yaml#
+      - $ref: /schemas/power/supply/qcom,pmi8998-fg.yaml#
 
   "^charger@[0-9a-f]+$":
     type: object

